client/locations-list-map.js

```diff
@@ -2051,6 +2054,8 @@ $(() => {
   initContactModal();
   initGlobalAlert();
 
+  // Get the relevant json data file based on country and requester/maker from findthemasks.com and
```

Comment:
we refer to "requester/maker" as a "dataset" throughout the codebase. I think that would be good to call out.

Reply:
Gotcha, I'll make that change on the next commit!